### ISSUE TITLE
Advice

### ISSUE THREAD

**Issue by elventevil** [Wolf49406/Dota2Patcher#1]
Hello, this is really usefull cheat thank you for that.
Im just curious, can u add VisiblebyEnemy in this cheat.it will be perfect all we need camera distance,fog and visiblebyenemy for deward wards :)

**Comment by Wolf49406** [comment]
Nah. You'll have to inject dll and bypass VAC. I have VisibleByEnemy in my other cheat, and probably i can write little external hack later.Nah. You'll have to inject dll and bypass VAC. I have VisibleByEnemy in my other cheat, and probably i can write little external hack later.
